README.md

```diff
@@ -0,0 +1,95 @@
+Aloha, developer!
+
+Alohalytics statistics engine solves following problems:
+- Track any event from iOS/Android/C++ code (e.g. events from Java and native C++ code will be collected perfectly).
+- Track basic events like iOS/Android app installs/updates/launches.
+- Track basic devices information including IDFA and Google Play Advertising IDs.
+- Collect all events while device is offline.
+
+More features are coming soon!
+
+If you have any questions, please contact Alex Zolotarev <me@alex.bio> from Minsk, Belarus.
+
+iOS
+======
+Minimum supported iOS version is 5.1.
+
+If you target iOS 5.1, then add an optional AddSupport.framework to your binary to access IDFA (for iOS 6+ you can add it as 'required').
+
+If your app uses background fetch, you can use ```[Alohalytics forceUpload];``` from ```application:performFetchWithCompletionHandler``` in your app delegate to improve events delivery.
+
+App updates are detected by changing CURRENT_PROJECT_VERSION in CFBundleShortVersionString key in the plist file.
+
+Built-in logged events
+------
+- $install
+- $launch
+- $update
+- $iosDeviceIds
+- $iosDeviceInfo
+- $browserUserAgent
+- $applicationDidBecomeActive
+- $applicationWillResignActive
+- $applicationWillEnterForeground
+- $applicationDidEnterBackground
+- $applicationWillTerminate
+
+
+Android
+======
+Minimum supported Android version is 2.3 (API level 9).
+
+Built-in logged events
+------
+- $install
+- $launch
+- $update
+- $androidIds
+- $androidDeviceInfo
+
+
+nginx server setup example
+======
+```
+http {
+  log_format alohalytics  '$remote_addr [$time_local] "$request" $status $content_length "$http_user_agent" $content_type $http_content_encoding';
+  server {
+    listen 8080;              # <-- Change to actual server port.
+    server_name localhost;    # <-- Change to actual server name.
```

Comment:
localhost is for testing purposes only, that’s why I put a visible comment on the right.

> On Jun 17, 2015, at 20:51, Dima <notifications@github.com> wrote:
> 
> In README.md <https://github.com/KnowSheet/Alohalytics/pull/37#discussion_r32654552>:
> 
> > +------
> > +- $install
> > +- $launch
> > +- $update
> > +- $androidIds
> > +- $androidDeviceInfo
> > +
> > +
> > +nginx server setup example
> > +======
> > +```
> > +http {
> > +  log_format alohalytics  '$remote_addr [$time_local] "$request" $status $content_length "$http_user_agent" $content_type $http_content_encoding';
> > +  server {
> > +    listen 8080;              # <-- Change to actual server port.
> > +    server_name localhost;    # <-- Change to actual server name.
> Then localhost is a bad example. LGTM otherwise.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/KnowSheet/Alohalytics/pull/37/files#r32654552>.
>